usr/include/slotmgr.h

```diff
@@ -33,10 +33,32 @@
 #define OCK_HSM_MK_CHANGE_PATH CONFIG_PATH "/HSM_MK_CHANGE"
 #define OCK_HSM_MK_CHANGE_LOCK_FILE LOCKDIR_PATH "/LCK..HSM_MK_CHANGElock"
 
-#define PROC_SOCKET_FILE_PATH "/run/opencryptoki/pkcsslotd.socket"
-#define ADMIN_SOCKET_FILE_PATH "/run/opencryptoki/pkcsslotd.admin.socket"
+#if defined(_AIX)
+   /*
+    * AIX needs the file to be opened in write mode for an exclusive lock.
+    * Mode bits are changed to -w--w---- to allow starting up even when there
+    * is a lock file left over from a previous start.
+    */
+    #define MODE_BITS (S_IWUSR | S_IWGRP)
+    #define OPEN_MODE O_WRONLY
+#else
+   /*
+    * Linux *does* have tmpfs, so it makes sense to use that for
+    * tmpfs-qualified files, such as runtime sockets and PID files. There is
+    * no use persisting those files after a reboot. Use /var/run here instead
```

Comment:
I guess this comment is a bit outdated. We ARE using the 'modern' /run on Linux. Please update acordingly.

Reply:
Ah, I guess I missed out removing this text altogether. I've already updated the commit message to explain the reasoning behind the `runstatedir` changes. I will get rid of this commit message, it is redundant.

Reply:
It seems you accidentally added a blank line in slotmgr.h line 326 while removing the comment....

Reply:
Which hunk are you referring to? There's no hunk for slotmgr.h that adds a blank line anywhere - I dropped a hunk that _removed_ a blank line to avoid unneeded whitespace modifications. Line 326 is the same as the equivalent line on master now.

Reply:
This is the hunk that I dropped:
```patch
@@ -299,5 +323,4 @@ static inline int ock_snprintf(char *buf
     return 0;
 }
 
-
 #endif                          /* _SLOTMGR_H */
```

Reply:
I am looking at the comparison from your last force push: https://github.com/opencryptoki/opencryptoki/compare/67b46fa0214e215562a0ef94a7ec276ec61ce06c..dd8402618a330cf67f4775a650e3e8a2648b08be 
Its this what I am referring to:
![image](https://github.com/opencryptoki/opencryptoki/assets/28868064/b6d0fb99-a946-41cc-8fc3-e1395ca34af8)

Reply:
Yeah, that's just the reversal of the line removal I'd done before. It's the effect of the hunk above that I dropped.